follina.py

```diff
@@ -6,6 +6,13 @@
 import socketserver
 import base64
 
+const_src_path = "src"
+const_src_docx_path = f"{const_src_path}/clickme"
+const_rtf_payload = "XpayloadX"
```

Comment:
Where is this used? The RTF file contains the same template as the docx, right? (`{payload_url}`)